webapp/app/country/actions.js

```diff
@@ -44,6 +45,16 @@ export const fetchCountryList = () => async (dispatch, getState) => {
   dispatch({ type: listCountries, countries })
 }
 
+export const fetchRegionList = () => async (dispatch, getState) => {
+  const { data: regions } = await axios.get('/api/country/regions')
+  const i18n = AppState.getI18n(getState())
+  const _getListname = (regionCode) => i18n.t(`area.${regionCode}.listName`)
+
+  regions.sort((region1, region2) => (_getListname(region1) > _getListname(region2) ? 1 : -1))
```

Comment:
sorting should happen in the Component where the regions are used.
And when user change language, they should be re-sorted